ui/src/hooks/useFileManage.ts

```diff
@@ -492,9 +502,6 @@ const generateUploadChunks = async (
  */
 const interraptUpload = (socket: WebSocket, fileInfo: UploadFileInfo) => {
   isStop.value = true;
-
-  console.log('fileInfo', fileInfo);
-  // TODO 增加后端标识
 };
 
 /**
```

Comment:
Your response has been cut off at "window.URL." The conversation seems to have ended abruptly. Please provide more context or complete your code for me to assist with specific queries related to the issues mentioned above.